db/index.js

```diff
@@ -63,12 +65,17 @@ function insert_transaction(
   category_id,
   vendor,
   amount,
+  file_name,
   callback
 ) {
+  const fs = require('fs');
+  if (!fs.existsSync(file_name)) {
+    file_name = null;
```

Comment:
I think we should have this throw an error if they try to upload a file that doesn't exist